src/Overlay/Dialog/Dialog.tsx

```diff
@@ -11,6 +11,7 @@ type DialogProps = {
     isOpen: boolean;
     onClose?: () => void;
     classNames?: ClassNames;
+    size?: string;
```

Comment:
The size prop type needs to be more specific `"sm" | "md" | "lg" | "full"`

Reply:
Updated.